files/usr/share/jeos-firstboot/raspberrywifi

```diff
@@ -56,7 +56,24 @@ is_raspberry()
 # interaction and access to the global systemd_firstboot_args array
 raspberrywifi_systemd_firstboot()
 {
-	if is_raspberry && ls -d /sys/class/net/*/wireless &>/dev/null; then
+        for p in /sys/class/net/* ; do
```

Comment:
`p` and `d` aren't great variable names

Reply:
just copied and pasted from the code on /lib/jeos-firstboot, gonna change in both places.